### DISCUSSION TITLE
Possible confusion about the two Change speed dialogs

### DISCUSSION THREAD

**Discussion by DavidBailes** [audacity/audacity#5248] [Development]
There are now (or will be very very soon) two change speed dialogs. A new dialog which acts non-destructively on a clip, and the old one for an effect which acts destructively on the selected audio.
The new dialog changes speed whilst preserving pitch, whereas the old dialog doesn't preserve pitch.
Users are used to the idea that "changing speed" doesn't preserve pitch, but there is nothing in the new dialog, or the commands that open it to suggest that pitch is preserved.
Wouldn't it be helpful to the user to make that explicit in some way? - text in the dialog, dialog title, etc?

**Comment by LWinterberg** [comment]
I'm not a big fan of using the near-synonymous words speed and tempo to distinguish between whether or not pitch will also be affected. 

How about we rename the "Change Speed" effect to "**Change Speed and Pitch**", so it's clear that both parts are affected? cc @DilsonsPickles @Tantacrul @petersampsonaudacity 

(to be clear, we're talking about this guy:
![image](https://github.com/audacity/audacity/assets/87814144/e5bc1533-a260-4413-bdd4-066223bbfc7a)
)

**Reply by Tantacrul** [reply]
Works for me.

**Comment by DavidBailes** [comment]
> Works for me.

and for me.

**Comment by crsib** [comment]
What about `Effect > Pitch and Tempo > Change Speed...`?

**Reply by crsib** [reply]
I mean, there are both "Change Speed" and "Change Pitch"

**Reply by LWinterberg** [reply]
![image](https://github.com/audacity/audacity/assets/87814144/6cdc8b42-9812-4a4a-85b0-1200c930c966)
that's the exact thing I'm renaming :)

**Comment by petersampsonaudacity** [comment]
@LWinterberg @crsib 

I have placed P1s in the Manual for this - but is not yet in master for 3.4.0

**Comment by petersampsonaudacity** [comment]
I have updated the Manual for this
